examples/next/package.json

```diff
@@ -18,6 +18,7 @@
     "@types/react": "18.2.8",
     "@types/react-dom": "18.2.4",
     "@unocss/postcss": "latest",
+    "@unocss/preset-attributify": "latest",
```

Comment:
You are use it import from `unocss`, why need install it again?

Reply:
It is quoted here, but it is not installed, so it does not need to be installed?

<img width="717" alt="image" src="https://github.com/unocss/unocss/assets/53572196/4682d648-491c-48f9-92a2-c1b020b8dee7">